package.yaml

```diff
@@ -165,6 +164,7 @@ executables:
           buildable: false
         else:
           buildable: true
+          dependencies: pretty-show
```

Comment:
Apparently not.  I tried it the other way and it didn't work.  The `pretty-show` package does not build with GHC, so I had to move it here.  This might be because `cabal2nix` does not take build flags into account.

Reply:
It doesn't build with GHC, or GHCJS?

I'll use your PR to reproduce the build here. There has to be a way to get this working that's sane; moving dependencies under a `Buildable` guard feels like too much of a hack.

Reply:
GHCJS

Reply:
@mightybyte I just tried merging this branch but undoing your `package.yaml`/`hnix.cabal` changes, and everything still built fine using `ghcjs.nix`. What do I need to do to see the error you ran into that motivated this change?

Reply:
Weird, I don't know.  I could swear it wasn't building for me without that, but I'm not sure what would be different.  The only thing I can think of is that maybe I made the `hnix` code changes first and then subsequently made changes to the hnix-web-repl `default.nix` file that made those changes irrelevant.  I'll give it a try with the new hnix master.